node/chain_spec/src/lib.rs

```diff
@@ -1143,18 +1147,28 @@ fn testnet_genesis(
     ];
     let initial_synthetic_assets = vec![XSTUSD.into()];
     GenesisConfig {
+        #[cfg(feature = "wip")]
```

Comment:
```suggestion
        #[cfg(feature = "wip")] // bridge
```

Reply:
Or other name.
Please use the comment for all your flags.
It makes more easy to recognize the feature scope of every single flag

Reply:
Added everywhere except imports